huxley/www/js/components/RegistrationView.js

```diff
@@ -90,12 +92,20 @@ var RegistrationView = React.createClass({
   },
 
   componentDidMount: function() {
-    CountryStore.getCountries(function(countries) {
-      this.setState({countries: countries});
-    }.bind(this));
-    CommitteeStore.getSpecialCommittees(function(committees) {
+    this._committeesToken = CommitteeStore.addListener(() => {
```

Comment:
I'm very much looking forward to getting all of our Stores onto standard Flux so that we can use FluxContainer to get rid of all this boilerplate.